pkg/validation/internal/bundle.go

```diff
@@ -137,14 +138,19 @@ func validateBundleSize(bundle *manifests.Bundle) []errors.Error {
 	var errs []errors.Error
 
 	if bundle.CompressedSize == nil || *bundle.CompressedSize == 0 {
-		errs = append(errs, errors.WarnFailedValidation("unable to check the bundle size", nil))
+		errs = append(errs, errors.WarnFailedValidation("unable to check the bundle compressed size", bundle.Name))
+		return errs
+	}
+
+	if bundle.Size == nil || *bundle.Size == 0 {
+		errs = append(errs, errors.WarnFailedValidation("unable to check the bundle size", bundle.Name))
 		return errs
 	}
 
 	if *bundle.CompressedSize > max_bundle_size {
-		errs = append(errs, errors.ErrInvalidBundle(fmt.Sprintf("maximum bundle compressed size with gzip size exceeded: size=~%d MegaByte, max=%d MegaByte", *bundle.CompressedSize/(1<<(10*2)), max_bundle_size/(1<<(10*2))), nil))
+		errs = append(errs, errors.ErrInvalidBundle(fmt.Sprintf("maximum bundle compressed size with gzip size exceeded: size=~%d MegaByte, max=%d MegaByte. Bundle uncompressed size is %d KB", *bundle.CompressedSize/(1<<(10*2)), max_bundle_size/(1<<(10*2)), *bundle.Size/int64(1024)), bundle.Name))
```

Comment:
maybe we could append the name of the bundle to the start of the error instead of the end so it's more clear, WDYT?
```suggestion
		errs = append(errs, errors.ErrInvalidBundle(fmt.Sprintf("maximum bundle compressed size with gzip size exceeded: bundle=%s, size=~%d MegaByte, max=%d MegaByte. Bundle uncompressed size is %d KB", bundle.Name, *bundle.CompressedSize/(1<<(10*2)), max_bundle_size/(1<<(10*2)), *bundle.Size/int64(1024))))
```

Reply:
We need to pass the name of the bundle otherwise it will be output as `Error : :` instead.